modules/ROOT/assets/attachments/sg.yaml

```diff
@@ -205,25 +205,34 @@ properties:
                     description: |
                       Enterprise Edition.
 
-                      High watermark for channel cache eviction (percent)
+                      High watermark for channel cache eviction (percent).
+
+                      When the cache size, determined by `max_number`, reaches the high watermark, the eviction process iterates through the cache to remove inactive channels.
                     default: 80
                   compact_low_watermark_pct:
                     type: integer
                     description: |
                       Enterprise Edition.
 
-                      Low watermark for channel cache eviction (percent)
+                      Low watermark for channel cache eviction (percent).
+
+                      When the cache size, determined by `max_number` returns to a value lower than `compact_low_watermark_pct`, the cache eviction process is stopped.
                     default: 60
                   max_number:
                     type: integer
                     description: |
                       Enterprise Edition.
 
-                      Maximum number of channel caches which will exist at any one point.
+                      Maximum number of channel caches which will exist at any one point. This property is used to determine the cache size (and the associated eviction watermarks `compact_low_watermark_pct`/`compact_high_watermark_pct`).
+
+                      The default value for this property is 50000. Along with the default channel `min_length` and `max_length` values, this would result in a memory usage under 1GB.
+
+                      The `max_number` value should be higher than the estimated number of replicating clients to optimize cache usage and minimize the number of cache misses.
 
-                      It should be set to equal the estimated number of replicating clients to avoid the impact of cache misses.
+                      If the cache size grows to reach the high watermark (`compact_high_watermark_pct`), channels with no connected replications will be evicted before channels which are associated with an active pull replication (i.e a blip-based pull replication in Couchbase Lite 2.x, or an active `/{db}/_changes` request in Couchbase Lite 1.x).
 
                       The minimum allowed value is 1.
+                      Setting the value to zero removes the limit to the number of channels being cached.
```

Comment:
A few corrections here:
- The minimum allowed value is 100
- It's not possible to set the value to zero.  Users who want to 'remove the limit' would need to set this to an arbitrarily high value - there's no built-in option to revert to 2.5-like behaviour.